iib/workers/tasks/build_merge_index_image.py

```diff
@@ -440,43 +440,73 @@ def is_bundle_version_valid(
     try:
-        bundle_version_label = get_image_label(bundle_path, 'com.redhat.openshift.versions')
-        if allow_no_ocp_version and not bundle_version_label:
+        bundle_version_label_tmp = get_image_label(bundle_path, 'com.redhat.openshift.versions')
+        if allow_no_ocp_version and not bundle_version_label_tmp:
             log.info(
                 'Marking bundle %s without label `com.redhat.openshift.versions` as valid.',
                 bundle_path,
             )
             return True
-        # MYPY error: Item "None" of "Optional[str]" has no attribute "replace"
-        bundle_version = bundle_version_label.replace('v', '')  # type: ignore
-        log.debug(f'Bundle version {bundle_version}, Index image version {valid_ocp_version}')
-        # MYPY error: Item "None" of "Optional[str]" has no attribute "startswith"
-        if bundle_version_label.startswith('='):  # type: ignore
-            if Version(bundle_version.strip('=')) == ocp_version:
-                return True
-        # MYPY error: Unsupported right operand type for in ("Optional[str]")
-        elif '-' in bundle_version_label:  # type: ignore
-            min_version, max_version = [Version(version) for version in bundle_version.split('-')]
-            if min_version <= ocp_version <= max_version:
-                return True
-        # MYPY error: Unsupported right operand type for in ("Optional[str]")
-        elif "," in bundle_version_label:  # type: ignore
-            versions = [Version(version) for version in bundle_version.split(",")]
-            if versions[0] <= ocp_version:
+
+        if '|' in bundle_version_label_tmp:
+            bundle_version_labels = bundle_version_label_tmp.split('|')
+        else:
+            bundle_version_labels = [bundle_version_label_tmp]
+
+        for bundle_version_label in bundle_version_labels:
+            # MYPY error: Item "None" of "Optional[str]" has no attribute "replace"
+            bundle_version = bundle_version_label.replace('v', '')  # type: ignore
+            log.debug(f'Bundle version {bundle_version}, Index image version {valid_ocp_version}')
+            # MYPY error: Item "None" of "Optional[str]" has no attribute "startswith"
+            if bundle_version_label.startswith('='):  # type: ignore
+                if Version(bundle_version.strip('=')) == ocp_version:
+                    return True
+            # MYPY error: Unsupported right operand type for in ("Optional[str]")
+            elif '-' in bundle_version_label:  # type: ignore
+                min_version, max_version = [
+                    Version(version) for version in bundle_version.split('-')
+                ]
+                if min_version <= ocp_version <= max_version:
+                    return True
+            # MYPY error: Unsupported right operand type for in ("Optional[str]")
+            elif bundle_version_label.startswith('>='):  # type: ignore
```

Comment:
this approach would also make only `>=4.15` also valid, no?

Reply:
I don't think only `>=v4.15` works in Pyxis (https://pyxis.engineering.redhat.com/v1/operators/indices?ocp_versions_range=%3E=v4.11) and neither does `<=v4.x` anymore. I believe the support for the latter ] was dropped some time ago.